### PR TITLE
NXP-24997: Add possibility to delegate and reassign a task

### PR DESCRIPTION
**Pull request by nmpcunha**

### CODE REVIEW

File: elements/workflow/nuxeo-document-task.html [hunk 1]
```diff
@@ -45,24 +46,53 @@
...
+        @apply --layout-vertical;
+      }
+
+      .horizontal {
+        @apply --layout-horizontal;
+      }
+
+      h3 {
+        margin-bottom: 0;
+        text-shadow: -0.1px 0 rgba(0, 0, 0, 0.4), 0 0.1px rgba(0, 0, 0, 0.4), 0.1px 0 rgba(0, 0, 0, 0.4), 0 -0.1px rgba(0, 0, 0, 0.4);
```
Comment (Gabez0r) [inline elements/workflow/nuxeo-document-task.html]: We should be using `font-weight: bold` here instead, but it won't work. Created [NXP-25590](https://jira.nuxeo.com/browse/NXP-25590) to address the issue.

File: elements/workflow/nuxeo-document-task.html [hunk 2]
```diff
@@ -146,7 +202,7 @@ <h3>[[i18n(task.name)]]</h3>
       },
 
       _processTask: function(e) {
-        if (this.validate()) {
+        if (this.validate('task-container')) {
```
Comment (Gabez0r) [inline elements/workflow/nuxeo-document-task.html]: Why is 'task-container' being passed as a parameter? The `validate` method does not receive any parameters.

Comment (nmpcunha) [reply elements/workflow/nuxeo-document-task.html]: It is incorrect, due to some code that was not completely reverted.
Thanks!

File: elements/workflow/nuxeo-document-task-assignment-popup.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+          type: Object
+        },
+
+        action: {
+          type: String
+        }
+      },
+
+      togglePopup: function() {
+        this._resetPopup();
```
[the remark this answers is not in the document]
Comment (nmpcunha) [reply elements/workflow/nuxeo-document-task-assignment-popup.html]: Will change a little bit the approach, thanks!

**Comment by Gabez0r** [comment]
👍

**Comment by guirenard** [comment]
:+1:

**Comment by nmpcunha** [comment]
https://qa.nuxeo.org/jenkins/view/TestAndPush/job/web-ui-pipeline-ncunha/8/

**Pull request merged**